R/get_manta_sv.R

```diff
@@ -50,6 +52,7 @@
 get_manta_sv = function(these_sample_ids,
                         these_samples_metadata,
                         projection = "grch37",
+                        this_seq_type = "genome",
```

Comment:
The version of this function in GAMBLR.results doesn't have this parameter - but maybe it will be introduced there as well and that change is coming?